roles/mediawiki/tasks/deprovision.yml

```diff
@@ -1,35 +1,35 @@
 ---
   - name: Removing endpoint
     k8s_v1_endpoints:
-      name: mediawiki123
+      name: mediawiki
```

Comment:
The version is going to be changing more frequently it seems. We need to get to 1.31 when it's released soon, so I'd rather just not incorporate a version. Even if we set it in the playbook/role there are other places to consider (CI tasks, Dockerfile, etc. etc.).

Reply:
What version? I was suggesting that you put in `defaults` or `vars` a new variable called `app_name: 'mediawiki'` and, here for example have:

```
k8s_v1_endpoints:
    name: {{ app_name }}
```

and so on throughout all the objects so that it would be more future proof.

Reply:
Dropped your quotes.